# Patch to clear the focus when the engine is reset. When it was not cleared
# it could cause crashes
#
# Copyright (C) 2006 Nokia Corporation.
# This file is distributed under the terms of GNU LGPL license, either version 2
# of the License, or (at your option) any later version.
#
--- gtkhtml-3.9.1/src/htmlengine.c~	2006-08-23 10:46:46.450790827 +0300
+++ gtkhtml-3.9.1/src/htmlengine.c	2006-08-23 10:47:10.760372776 +0300
@@ -5019,6 +5019,8 @@
 
 	e->flow = NULL;
 
+	e->focus_object = NULL;
+
 	/* reset to default border size */
 	e->leftBorder   = LEFT_BORDER;
 	e->rightBorder  = RIGHT_BORDER;
